SPHINXsys/src/shared/io_system/in_output.h

```diff
@@ -310,6 +310,22 @@ namespace SPH
 		VtuStringData _vtuData;
 	};
 
+	/**
+	 * @class SurfaceOnlyBodyStatesRecordingToVtu
+	 * @brief  Write files for surface particles of bodies
+	 * the output file is VTK XML format can visualized by ParaView
+	 * the data type vtkUnstructedGrid
+	 */
+	class SurfaceOnlyBodyStatesRecordingToVtu : public BodyStatesRecording
```

Comment:
This was added to upstream commit faa88cab07866f9ba5e370654ab3b515c46c6432 from our side
Removed from upstream during merge in PR #60 
Passed through during this merge, yielding duplicate symbols.